spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/env/EnvironmentEndpoint.java

```diff
@@ -195,8 +206,8 @@ private void extract(String root, Map<String, PropertySource<?>> map,
 		}
 	}
 
-	public Object sanitize(String name, Object object) {
-		return this.sanitizer.sanitize(name, object);
+	private Object sanitize(PropertyNameAndSource source, Object object) {
```

Comment:
Couldn't find out why this method was public, made it private since I changed the signature anyway. If we make this public again then `PropertyNameAndSource` must have a public constructor which it now doesn't.